src/docs/Components/Form/PasswordInput/index.mdx

```diff
@@ -17,7 +17,7 @@ status:
 
 > Password input is a necessary security feature for logging in or registering on a website.
 
-Place preview here !
+<Preview path="default" nude />
```

Comment:
@tiloyi Can you prefill the input with a password? For example "Password1234".

Reply:
Fixed